tools/deps/deps_darwin

```diff
@@ -47,4 +47,4 @@ packages=(
 )
 
 function install_package() { brew install $1; }
-function query_package() { brew list | grep -q $1; }
+function query_package() { brew list | { grep -q $1; cat > /dev/null;} }
```

Comment:
`cat > /dev/null` to resolve `SIGPIPE` would overwrite the exit status from grep. So, store the exit status of grep and return that.

Reply:
This should be fixed now!